app/src/main/java/app/revanced/integrations/patches/theme/SeekbarColorPatch.java

```diff
@@ -51,12 +51,31 @@ public static int getCustomSeekbarColor() {
         return customSeekbarColor;
     }
 
+
+    /**
+     * Injection point.
+     *
+     * Overrides all Litho components that use the YouTube seekbar color.
+     * Used only for the seekbar shown in view thumbnails.
+     *
+     * If {@link SettingsEnum#HIDE_SEEKBAR_THUMBNAIL} is enabled, this returns a fully transparent color.
+     */
+    public static int getLithoColorOverride(int colorValue) {
+        if (colorValue == ORIGINAL_SEEKBAR_COLOR) {
+            if (SettingsEnum.HIDE_SEEKBAR_THUMBNAIL.getBoolean()) {
+                return 0x00000000;
+            }
+            return getSeekbarColorValue(ORIGINAL_SEEKBAR_COLOR);
+        }
+        return colorValue;
+    }
```

Comment:
I believe the methods were separated so that the niche option to color only while the player is focused exists. It may be refactored back to it's original implementation according to https://github.com/revanced/revanced-patches/pull/2252#issuecomment-1562808600.